spec/lib/catalog/rbac/access_spec.rb

```diff
@@ -59,32 +62,12 @@
       let(:rbac_enabled) { true }
 
       before do
-        allow(catalog_access).to receive(:scopes).with("portfolio_items", verb).and_return(scopes)
+        allow(catalog_access).to receive(:scopes).with(aceable_type.table_name, verb).and_return(scopes)
       end
 
       context "when the user is in the group scope" do
         let(:scopes) { %w[group user] }
 
-        let(:group_pagination) do
```

Comment:
Was this unused before and was removed in this PR?

Reply:
Yeah, I threw this in here because it helps to clean up the example and separate out the noise/setup from the meat of it. It could probably be moved to a separate PR, but I don't think it hurts this one.